src/main/java/carpetextra/CarpetExtraSettings.java

```diff
@@ -281,4 +281,11 @@ public Integer validate(ServerCommandSource source, CarpetRule<Integer> currentR
             categories = {EXTRA, FEATURE, DISPENSER}
     )
     public static boolean fallingBlockDispensers = false;
+
+    @Rule(
+            options = {"5", "10", "20"},
+            strict = false,
+            categories = {EXTRA, FEATURE}
```

Comment:
Suggestion based on the feedback above:
```suggestion
            categories = {EXTRA, FEATURE},
            validators = Validators.NonNegativeNumber.class
```
And the import for `Validators`.